themes/haystack/layouts/partials/tutorial-card.html

```diff
@@ -0,0 +1,36 @@
+<a href="{{ .RelPermalink }}" class="tutorial-card" data-level="{{ .Params.level }}" data-date="{{ .Params.last_updated }}" data-task="{{ .Params.task }}"  {{ with .Params.new }}data-new="{{ . }}"{{ else }}data-new="false"{{ end }}>
+    {{/*  Header / tags  */}}
+    <div class="tutorial-card-header">
+        {{/*  level  */}}
+        <div>
+            <span class="icon"><svg width="10" height="11" viewBox="0 0 10 11" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M9.64185 4.13731L6.66646 3.70489L5.33638 1.0084C5.30005 0.934575 5.24029 0.874809 5.16646 0.838481C4.9813 0.747075 4.7563 0.823247 4.66372 1.0084L3.33365 3.70489L0.358256 4.13731C0.276224 4.14903 0.201224 4.1877 0.143802 4.24629C0.0743826 4.31764 0.0361291 4.41364 0.0374475 4.51318C0.038766 4.61272 0.0795484 4.70767 0.150834 4.77715L2.30357 6.87598L1.79497 9.83965C1.78305 9.90859 1.79068 9.9795 1.817 10.0443C1.84332 10.1092 1.88727 10.1653 1.94388 10.2064C2.00049 10.2475 2.06749 10.272 2.13728 10.2769C2.20707 10.2819 2.27685 10.2672 2.33872 10.2346L5.00005 8.83536L7.66138 10.2346C7.73404 10.2732 7.81841 10.2861 7.89927 10.2721C8.10318 10.2369 8.24029 10.0436 8.20513 9.83965L7.69654 6.87598L9.84927 4.77715C9.90787 4.71973 9.94654 4.64473 9.95826 4.5627C9.9899 4.35762 9.84693 4.16778 9.64185 4.13731ZM6.79068 6.58067L7.21373 9.04512L5.00005 7.88262L2.78638 9.04629L3.20943 6.58184L1.4188 4.83575L3.8938 4.47598L5.00005 2.23418L6.1063 4.47598L8.5813 4.83575L6.79068 6.58067Z" fill="#2B2F55"/>
+            </svg></span>
+            <span>{{ .Params.level }}</span>
+        </div>
+
+        {{/*  Completion time  */}}
+        <div>
+            <span class="icon">
+                <svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M6 0.75C3.10078 0.75 0.75 3.10078 0.75 6C0.75 8.89922 3.10078 11.25 6 11.25C8.89922 11.25 11.25 8.89922 11.25 6C11.25 3.10078 8.89922 0.75 6 0.75ZM6 10.3594C3.59297 10.3594 1.64062 8.40703 1.64062 6C1.64062 3.59297 3.59297 1.64062 6 1.64062C8.40703 1.64062 10.3594 3.59297 10.3594 6C10.3594 8.40703 8.40703 10.3594 6 10.3594Z" fill="#2B2F55"/>
+                    <path d="M8.04726 7.48358L6.37617 6.27538V3.375C6.37617 3.32344 6.33398 3.28125 6.28242 3.28125H5.71875C5.66719 3.28125 5.625 3.32344 5.625 3.375V6.60233C5.625 6.6328 5.63906 6.66093 5.66367 6.67851L7.60195 8.09178C7.64413 8.12225 7.70273 8.11288 7.7332 8.07186L8.06835 7.61483C8.09882 7.57147 8.08945 7.51288 8.04726 7.48358Z" fill="#2B2F55"/>
+                </svg>
+            </span>
+            
+            {{/*  <span>{{ .Params.completionTime }}</span>  */}}
```

Comment:
can we make this `completion_time` so that it's similar to the rest of the frontmatter arguments?
And is it possible to make it so that it's added _if_ the competion time is there and if not, nothing? 
Just so that we can still use it while some tutorials don't have completion times :)